### ISSUE TITLE
Solution to, Error loading "D:\AI\ai-toolkit\venv\Lib\site-packages\torch\lib\fbgemm.dll"

### ISSUE THREAD

**Issue by Hikska** [ostris/ai-toolkit#133]
Hi,
I just wanted to share
While following the instructions i got this error, when launching the training
```
{...]
  File "D:\AI\ai-toolkit\jobs\process\__init__.py", line 1, in <module>
    from .BaseExtractProcess import BaseExtractProcess
  File "D:\AI\ai-toolkit\jobs\process\BaseExtractProcess.py", line 4, in <module>
    from safetensors.torch import save_file
  File "D:\AI\ai-toolkit\venv\Lib\site-packages\safetensors\torch.py", line 6, in <module>
    import torch
  File "D:\AI\ai-toolkit\venv\Lib\site-packages\torch\__init__.py", line 148, in <module>
    raise err
Error loading "D:\AI\ai-toolkit\venv\Lib\site-packages\torch\lib\fbgemm.dll
```

this was solved by this solution: https://github.com/Acly/krita-ai-diffusion/issues/965

tldr, put the dll from the zip below into windows/system32
https://www.dllme.com/dll/files/libomp140_x86_64/00637fe34a6043031c9ae4c6cf0a891d/download

it fixed the problem instantly !

**Comment by Hikska** [comment]
i meant maybe adding a note, in case people have this problem could help

**Comment by EuphoricPenguin** [comment]
Anaconda is also worth a shot if people are installing from scratch on Windows; I haven't had any weird errors, and I'm also training LoRAs for Schnell.

**Comment by CambridgeComputing** [comment]
I had the same error. You need to go download Visual Studio 2022 Community Edition and just select the following option when installing:
![352517926-81707c5d-d56e-49ce-8677-460a6c423d62](https://github.com/user-attachments/assets/7dfe2676-df2b-4fa2-8283-d27586785c44)
